scripts/build-container.sh

```diff
@@ -4,10 +4,14 @@
 OLS_VERSION=v0.0.0
 
 # To build container for local use
-podman build --no-cache --build-arg=VERSION=${OLS_VERSION} -t ols:latest -f Containerfile
+if [ -z $OLS_NO_IMAGE_CACHE ]; then
+  podman build --no-cache --build-arg=VERSION=${OLS_VERSION} -t ${OLS_API_IMAGE:-quay.io/openshift/lightspeed-service-api:latest} -f Containerfile
+else
+  podman build --build-arg=VERSION=${OLS_VERSION} -t ${OLS_API_IMAGE:-quay.io/openshift/lightspeed-service-api:latest} -f Containerfile
+fi
 
 # To publish to a registry tag registry name
 # podman tag ols:latest quay.io/<org>/ols:$OLS_VERSION
```

Comment:
not sure what we want to do w/ this, in theory having OLS_API_IMAGE as a configurable env var should make this step unnecessary

Reply:
Correct, the version is not necessary. Is there a good way to encode the version in the container so that in troubleshooting we can check the version is the correct one and it is not the system using an older version of the image? (or should this be done by the Pod.spec definition and not on build)?

Reply:
OCP encodes git commits in the container labels, but i believe that's done as part of the build process, not something coded in each repo's dockerfile/build script.

take a look at

`docker inspect quay.io/openshift/lightspeed-service-api:latest`

has
```
                "io.k8s.description": "AI-powered OpenShift Assistant Service.",
                "io.k8s.display-name": "OpenShift LightSpeed Service",
                "io.openshift.build.commit.author": "",
                "io.openshift.build.commit.date": "",
                "io.openshift.build.commit.id": "29c08c736fea065f0e6faff557a326e49533716b",
                "io.openshift.build.commit.message": "",
                "io.openshift.build.commit.ref": "main",
                "io.openshift.build.name": "",
                "io.openshift.build.namespace": "",
                "io.openshift.build.source-context-dir": "",
                "io.openshift.build.source-location": "https://github.com/openshift/lightspeed-service",
```

that's probably good enough for now.